applications/sample/servlet/minifierAppSetup.json

```diff
@@ -1,410 +1,434 @@
 {
-	"startupSequence": [
```

Comment:
I can see that there's 24 lines added, but about 3000 lines worth of changes... Not too easy to see what's been added. I guess nothing should have been added as the new bundles are in another PR?